river/preprocessing/random_projection.py

```diff
@@ -0,0 +1,160 @@
+import collections
+import random
+import typing
+
+from river import base
+
+
+class GaussianRandomProjector(base.Transformer):
+    """Gaussian random projector.
+
+    This transformer reduces the dimensionality of inputs through Gaussian random projection.
+
+    The components of the random projections matrix are drawn from `N(0, 1 / n_components)`.
```

Comment:
Good stuff!! 

Is there a missing `** 0.5` in the docstring?

Reply:
Good catch :)